charts/fluent-bit/values.yaml

```diff
@@ -45,13 +45,16 @@ podSecurityPolicy:
   create: false
   annotations: {}
 
+# OpenShift-specific configuration
 openShift:
-  # Sets Openshift support
   enabled: false
-  # Creates SCC for Fluent-bit when Openshift support is enabled
   securityContextConstraints:
-    create: true
+    # Create SCC for Fluent-bit and allow use it
+    create: false
```

Comment:
Wouldn't we be better to default to creating a new SCC as that way this wouldn't fail without an additional input?

Reply:
@stevehipwell 
Yes, it makes sense. Despite of I'm thinking that OpenShift engineers and developers using OpenShift must know when to enable it :) But there are could be cases when inexperienced fellows will run Fluent-Bit via chart and facing issues without SCC creation enabled.